### DISCUSSION TITLE
diretory visusal studio windows 11

### DISCUSSION THREAD

**Discussion by InterConectDigital** [psake/psake#378] [Q&A]
Hello I use psake with visual studio on windows 10 and it works correctly, but when I use it on windows 11 it returns this error below. Does anyone know how to change visual studio path in psake

diretory vs windows 11 => C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.

![Screenshot_1](https://user-images.githubusercontent.com/53632931/157473026-132818ae-0f37-4e8b-b314-31c7dda507fd.png)
0

**Comment by HeyItsGilbert** [comment]
This was an artifact of the old in-repo help/build process. Help generation and packaging have since moved to PowerShellBuild, and the current build doesn't leave a .old file behind in en-US. Closing as resolved - if you still hit a stray .old in a fresh install from the gallery, let me know the version and I'll dig in.
